test/e2e/network/util.go

```diff
@@ -76,3 +79,27 @@ func newAgnhostPod(name string, args ...string) *v1.Pod {
 		},
 	}
 }
+
+// VerifySCTPModuleLoadedOnNodes checks whether any node on the list has the
+// sctp.ko module loaded
+func VerifySCTPModuleLoadedOnNodes(f *framework.Framework, nodes *v1.NodeList) bool {
```

Comment:
"Verify" makes it sound like the goal of the function is to ensure that SCTP *is* loaded, when in fact we want the opposite. Maybe rename to `CheckSCTPModuleLoadedOnNodes`? Or else `VerifySCTPModuleNotLoadedOnNodes` but then you'd have to flip the sense of the return value too.

Reply:
Fixed.